packages/roosterjs-editor-plugins/lib/plugins/CustomReplace/CustomReplace.ts

```diff
@@ -129,9 +143,8 @@ export default class CustomReplacePlugin implements EditorPlugin {
         const lowerCaseStringToSearch = stringToSearch.toLocaleLowerCase();
         for (const replacement of this.replacements) {
             const [sourceMatch, replacementMatch] = replacement.matchSourceCaseSensitive
-                ? [stringToSearch, replacement.sourceString]
+                ? [stringToSearch.replace(/\s/g, ' '), replacement.sourceString]
```

Comment:
Need to move this out of the loop, otherwise the replace will be executed many times, it is a waste